public/v4/apps/audiobookshelf.yml

```diff
@@ -0,0 +1,37 @@
+captainVersion: 4
+services:
+  $$cap_appname:
+    image: ghcr.io/advplyr/audiobookshelf:$$cap_version
+    ports:
+      - 13378:80
+    volumes:
+      - $$cap_path_to_audiobookshelf/config:/config
+      - $$cap_path_to_audiobookshelf/audiobooks:/audiobooks
+      - $$cap_path_to_audiobookshelf/podcasts:/podcasts
+      - $$cap_path_to_audiobookshelf/metadata:/metadata
+    restart: always
+
+caproverOneClickApp:
+  variables:
+    - id: "$$cap_version"
+      label: Audiobookshelf Version
+      defaultValue: "2.2.14"
+      description: Check out their Docker page for the valid tags https://hub.docker.com/r/advplyr/audiobookshelf/tags
+      validRegex: "/.{1,}/"
+
+    - id: $$cap_path_to_audiobookshelf
+      label: Location of app folder in host
+      description: path to base folder that will contain your audiobooks, podcasts, metadata and config.
```

Comment:
This is not clear to the end user. If this directory doesn't exist, the app is going to crash on start up. If any of the subdirectories don't exist, again, the app will crash.

Instead, I suggest just use volume name, and in instructions.end ask the user to remap the directories to where they have their content.